ports/stm32/boards/ARDUINO_GIGA/wifi_nvram_1dx.h

```diff
@@ -0,0 +1,49 @@
+static const uint8_t wifi_nvram_4343[] __attribute__((aligned(4))) =
```

Comment:
might be worth using `CYW43_RESOURCE_ATTRIBUTE` here

Reply:
Ah, yes, that's true.  But also you need the new cyw43-driver to get the alignment fix.

So, maybe add a commit that updates the driver?

Reply:
Updated the submodule in this PR.

Reply:
Now `cywbt.c` needs to include `cyw43_config.h` but the library is not the include path, I'm not sure how you want to fix it ? Maybe move `cywbt.c` to cyw43 library ?

Reply:
I included the config file for now, retested bt & wifi on the GIGA and Portenta still works fine.